File: app/src/main/java/ru/application/homemedkit/connectionController/models/VidalData.java
package ru.application.homemedkit.connectionController.models;

import static ru.application.homemedkit.helpers.ConstantsHelper.BLANK;

import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
import com.fasterxml.jackson.annotation.JsonProperty;
import com.fasterxml.jackson.annotation.JsonSetter;
import com.fasterxml.jackson.annotation.Nulls;

@JsonIgnoreProperties(ignoreUnknown = true)
public class VidalData {
    @JsonProperty("phKinetics")
    @JsonSetter(nulls = Nulls.SKIP)
    public String phKinetics = BLANK;
}

File: app/src/main/java/ru/application/homemedkit/connectionController/models/VidalData.kt
package ru.application.homemedkit.connectionController.models

import com.squareup.moshi.Json
import ru.application.homemedkit.helpers.BLANK

data class VidalData(
    @field:Json val phKinetics: String? = BLANK,
    @field:Json val images: List<String>? = emptyList()
)